papierowyRPG_API/Models/Stats.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Stats
+    {
+        public int ID { get; set; }
+        public Character Character { get; set; } = null!;
+        public Skills Skills { get; set; } = null!;
+        public Items Items{ get; set; } = null!;
+        public string[] stats { get; set; }
```

Comment:
Shouldn't it be values of stats? Like that?:
```suggestion
        public int[] Stats { get; set; }
```